test/cider-apropos-tests.el

```
@@ -1,20 +1,17 @@
-(require 'ert)
-(require 'noflet)
+(require 'buttercup)
 (require 'cider)
 (require 'cider-apropos)
 
-(ert-deftest cider-apropos-not-connected ()
-  (noflet ((cider-connected-p () nil))
-    (should-error (cider-apropos "test") :type 'user-error)))
+(describe "cider-apropos"
+  (it "raises user-error when cider is not connected."
+    (expect (lambda () (cider-apropos "test")) :to-throw 'user-error))
```

I believe it is necessary for the `to-throw` matcher. Otherwise the error is not handled and the tests fail with `error: (user-error "`nil' needs an active nREPL connection")`.